asmcli/lib/context.sh

```diff
@@ -101,3 +103,27 @@ context_append-kube-yaml() {
 context_list-kube-yamls() {
   jq -S -r '.kubectlFiles[]' "${context_FILE_LOCATION}"
 }
+
+context_append-clusters-info() {
+  local CLUSTER; CLUSTER="${1}"
+  local TEMP_FILE; TEMP_FILE="$(mktemp)"
+
+  jq -S --arg CLUSTER "${CLUSTER}" '.clustersInfo += [$CLUSTER]' "${context_FILE_LOCATION}" >| "${TEMP_FILE}" \
```

Comment:
It looks like you can pull some of this logic into a separate function and share it.

Reply:
Done